src/routes/welcome/container/index.tsx

```diff
@@ -1,16 +1,12 @@
 import * as React from 'react'
-import { connect } from 'react-redux'
-
-import Layout from '../components/Layout'
-
-import selector from './selector'
+import { WelcomeLayout } from 'src/routes/welcome/components/index'
```

Comment:
I've seen this in other files too, why are you being explicit about the `index` file?

Reply:
Just a warning from my IDE, I removed it